carbon-design-system/ibm-security
# feat(tagwall): apply id to each tag, allow extra attrs for tags

## Affected issues

- Resolves #530

## Proposed changes

- add the `id` from the `items` prop to each `Tag` inside `TagWall`
- add a new key to the `items` object to allow for extra attributes on specific tags
- update snapshots to reflect prop updates

## Testing instructions

- I updated the mocks to use the new `props` object so you should be able to see `example-attribute` rendered in the storybook demo.
